### PR TITLE
News bootstrap

### PR DESCRIPTION
**Pull request by jaimdedan**
This is a project of cloning Newsweek's webpage. We used the mobile-first philosophy and we created two breakpoints for tablet and desktop.
For this project, we decided to use Sass as a preprocessor to help us with color variables and mixins for different elements. And also we used bootstrap to help us style the project.

### CODE REVIEW

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Request changes 😳

I am very happy to see this project and I can see you worked hard on this project. There are only a few issues that we need to address and I am sure that you will end up with a perfect project

_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_

This is a great opportunity to learn. Microverse is here to help you. 

**Please have a look at the suggestions to get more understanding.**

- [x] Please remove linter errors. Always check if there is an error before submitting a review request. 


- [x] Please don't leave **alt** and **href** attributes empty. This is not a good practice. 
- [x] You have used too much CSS although the purpose of this project is to show your bootstrap skills. Please work on the changes and change CSS to bootstrap. 
- [x] Please add CSS, SCSS, and assets folders into a single folder. 


- [x] On the desktop version of your webpage the login and Subscribe button lacks proper stylings. Please work on them. 

**Yours Clone**


**Original**



- [x] The debate section is almost similar to the original one. But a few modifications required. 

**Clone**



**Original**



- [x] Please decrease the font-size for OPINION Section. 

- [ ] Please remove these empty spaces
1- 


2- 


- [x] Please remove the white space from IN `THE MAGAZINE` section. 

**Clone**



**Original**



- [x] Please decrease the white space from this section 


- [x] Please make these changes from the footer section

**Clone**



**Original**





**Please make the changes and request a re-review for the given project**

If you get stuck somewhere, don't hesitate to reach me out or schedule s 1:1 meeting with a Full-time TSE. 

Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,702 @@
...
+    padding-top: 1.5rem;
+    border-top: 1px solid #e2e2e2;
+  }
+  .us-news .card {
+    margin: 1.5rem 10px 0 0;
+  }
+
+  .subscribe-section .info-text {
+    max-width: 390px;
+    text-align: left !important;
```
Comment (arslanbisharat) [inline css/style.css:655]: - [ ] Please remove important

**Review by arslanbisharat — CHANGES_REQUESTED** [review]

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Request changes 😳

I am very happy to see this project and I can see you worked hard on this project. There are only a few issues that we need to address and I am sure that you will end up with a perfect project

_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_

This is a great opportunity to learn. Microverse is here to help you. 

**Please have a look at the suggestions to get more understanding.**

- [x] Please decrease the font-size for OPINION Section.

- [ ] Please remove these empty spaces from the area described in the previous review.

- [ ] Please remove the white space from `IN THE MAGAZINE` section.

- [x] YOUR CLONE IS MISSING MANY SECTIONS. PLEASE WORK ON THEM. I AM SORRY I MISSED THOSE IN MY PREVIOUS REVIEW. 

- [ ] Newsweek logo in footer is too much big


**Please make the changes and request a re-review for the given project**

If you get stuck somewhere, don't hesitate to reach me out or schedule s 1:1 meeting with a Full-time TSE. 

Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Comment by jaimdedan** [comment]
Fixed all the observations. Thanks a lot for your time.

**Comment by descholar-ceo** [comment]
# This Pull Request is under review by [descholar-ceo](https://github.com/descholar-ceo) :eyes: :eyes: 
## Please wait a bit ...

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Changes Required :vertical_traffic_light: :warning: 
## Hello @jpdf00 and @jaimdedan  :handshake: :handshake: :handshake: 

:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes from the previous review, keep it up, and your project is really almost done.
#### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
### Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 
- [x] Add this project name in your readme file, currently the title says `Project Name` :point_right: this is irrelavant
- [x] The Top stories should be at the top in mobile screens, currently at the top there is Featured stories, please fix that
- [x] Fix the subscribe section on both desktop screens and tablet screens (large and medium screens). Learn the screenshots below

| Your clone | Original page |
| --- | --- |
|Screenshot from 2020-12-11 04-08-02|Screenshot from 2020-12-11 04-08-48|

:point_right: Could you see the difference :question: :question: :question: 

- [x] You are missing vertical line after each link in the footer, and some spacing to make it look like the original. Fix them as well.
___
_If anything seems to be unclear to you, reachout and ask anything by commenting on this Pull Request and mention me as [descholar-ceo](https://github.com/descholar-ceo), I will receive a notification and will respond._

[GitHub](https://github.com/descholar-ceo) | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

_Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)_

**Comment by mimipeshy** [comment]
## STATUS : APPROVED 🥳 🍾 

Hello team @jaimdedan and @jpdf00, congratulations your project is approved! Go ahead and merge. All the best in the next project 😄 

Happy coding!

Regards
Peris Ndanu

**Pull request merged**